config/crd/bases/falcon.crowdstrike.com_falconconfigs.yaml

```diff
@@ -40,6 +40,10 @@ spec:
                 description: FalconAPI configures connection from your local Falcon
                   operator to CrowdStrike Falcon platform.
                 properties:
+                  cid:
+                    description: Falcon Customer ID (CID)
+                    pattern: '[0-9a-zA-Z]+-[0-9a-zA-Z]{2}'
```

Comment:
Can you make the regex `^[0-9a-fA-F]{32}-[0-9a-fA-F]{2}$` or `[0-9a-fA-F]{32}-[0-9a-fA-F]{2}`

Reply:
Yes, I can.